### ISSUE TITLE
GHC 9.6

### ISSUE THREAD

**Issue by andreasabel** [snapframework/snap#233]
Trying to build `snap` with GHC 9.6 it seems we are (at least) stuck at `clientsession`:
```
Error: cabal: Could not resolve dependencies:
[__0] trying: snap-1.1.3.2 (user goal)
[__1] trying: clientsession-0.9.1.2 (dependency of snap)
[__2] trying: crypto-random-0.0.9 (dependency of clientsession)
[__3] next goal: unix (dependency of crypto-random)
[__3] rejecting: unix-2.8.1.0/installed-2.8.1.0 (conflict: crypto-random =>
unix<2.8)
[__3] trying: unix-2.7.3
[__4] trying: cprng-aes-0.6.1 (dependency of clientsession)
[__5] trying: cereal-0.5.8.3 (dependency of snap)
[__6] next goal: bytestring (dependency of snap)
[__6] rejecting: bytestring-0.11.4.0/installed-0.11.4.0 (conflict: unix =>
base>=4.10 && <4.18, bytestring => base==4.18.0.0/installed-4.18.0.0)
[__6] rejecting: bytestring-0.12.0.0 (conflict: snap => bytestring>=0.9.1 &&
<0.12)
[__6] trying: bytestring-0.11.5.0
[__7] next goal: base (dependency of snap)
[__7] rejecting: base-4.18.0.0/installed-4.18.0.0 (conflict: unix =>
base>=4.10 && <4.18)
[__7] fail (backjumping, conflict set: base, snap, unix)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: base, bytestring, cprng-aes, snap,
unix, clientsession, cereal, crypto-random
```

The failure of `clientsession` with GHC 9.6 has already been brought to the attention of the maintainer:
- [x] https://github.com/yesodweb/clientsession/issues/37

**Comment by andreasabel** [comment]
I could build `snap` from `master` now on GHC 9.6.  
Maybe a release could be made now, @cydparser ?

**Comment by cydparser** [comment]
I've released [v1.1.3.2](https://hackage.haskell.org/package/snap-1.1.3.2)
